### PR TITLE
[Test] Increase test coverage by test Rector object ->getRuleDefinition() instanceof RuleDefinition

### PR DESCRIPTION
**Pull request by samsonasik**

### CODE REVIEW

**Comment by samsonasik** [comment]
All checks have passed 🎉 @TomasVotruba it is ready for review.

File: packages/testing/src/PHPUnit/AbstractRectorTestCase.php [hunk 1]
```diff
@@ -307,6 +308,12 @@ protected function doTestFileInfo(SmartFileInfo $fixtureFileInfo, array $extraFi
             return;
         }
 
+        $rectorClass = $this->getRectorClass();
+        if (self::$container->has($rectorClass)) {
+            $rectorObject = self::$container->get($rectorClass);
+            $this->assertInstanceOf(RuleDefinition::class, $rectorObject->getRuleDefinition());
+        }
+
```
Comment (TomasVotruba) [inline packages/testing/src/PHPUnit/AbstractRectorTestCase.php:316]: This test does not bring much value. This is tested by PHP itself as there is `RuleDefinition` return type

Comment (samsonasik) [reply packages/testing/src/PHPUnit/AbstractRectorTestCase.php:316]: ok 👍

**Comment by TomasVotruba** [comment]
What would be really helpful, is to improve the performance of tests. Now they take 40-50 seconds, because tests with custom `provideConfigFileInfo()` invoke new container rebuild. If there was a way to speed this up, it would save us CI and local times.

**Pull request closed without merging**